.gitignore

```diff
@@ -0,0 +1 @@
+git-crypt-key
```

Comment:
⭐ For not committing sensitive secrets (e.g. API keys) unencrypted to the repository.